### PR TITLE
Updating hotels (search) component

### PR DESCRIPTION
**Pull request by thomasthesecond**
This PR also includes updates to some mixins.

### CODE REVIEW

File: sass/_base_mixins.scss [hunk 1]
```diff
@@ -23,40 +34,36 @@
...
 
     max-width: $max-width;
     height: 0;
 
     border-bottom: .1rem solid $color-border;
 
-    @media (min-width: $min-720){
-      margin: 0 ($gutter*1);
+    @media (min-width: $min-720) {
+      margin-left: ($gutter*1);
```
Comment (jcreamer898) [inline sass/_base_mixins.scss]: I don't think this means what you think it means... $gutter * 1 === $gutter?

Comment (jcreamer898) [inline sass/_base_mixins.scss]: aka 30 * 1 === 30.

File: src/components/hotels/_hotels.scss [hunk 2]
```diff
@@ -1,221 +1,176 @@
 @import "../../../sass/webpack_deps";
 @import "../../../sass/datepicker";
 
-$small-breakpoint: 600;
-$mobile-breakpoint: 717;
-$wide-breakpoint: 1024;
-$form-height: 4.2rem;
+$hotels-breakpoint-small-max: em(729px);
```
Comment (jcreamer898) [inline src/components/hotels/_hotels.scss]: Is there no way to use the `$min-xxx` breakpoints from the `_variables`?

**Comment by jcreamer898** [comment]
Left a few comments, but otherwise... :shipit:

File: sass/_base_mixins.scss (hunk 1 above)
Comment (thomasthesecond) [inline sass/_base_mixins.scss]: Good catch. I think some of this was pre-existing but you're right, should just be `$gutter`.

File: src/components/hotels/_hotels.scss (hunk 2 above)
Comment (thomasthesecond) [inline src/components/hotels/_hotels.scss]: I can see if any of those breakpoints match up.

**Comment by jcreamer898** [comment]
Looks good! :+1:

**Pull request merged**